test/specs/rules/require-default-parameters.js

```diff
@@ -0,0 +1,38 @@
+var Checker = require('../../../lib/checker');
```

Comment:
Add test for `if` form:

```js
if (typeof b !== 'undefined')
```

and the other cases not covered by your `if` condition